src/init.cpp

```diff
@@ -746,6 +746,7 @@ void SetupServerArgs(NodeContext& node)
     argsman.AddArg("-llmq-qvvec-sync=<quorum_name>:<mode>", strprintf("Defines from which LLMQ type the masternode should sync quorum verification vectors. Can be used multiple times with different LLMQ types. <mode>: %d (sync always from all quorums of the type defined by <quorum_name>), %d (sync from all quorums of the type defined by <quorum_name> if a member of any of the quorums)", (int32_t)llmq::QvvecSyncMode::Always, (int32_t)llmq::QvvecSyncMode::OnlyIfTypeMember), ArgsManager::ALLOW_ANY, OptionsCategory::MASTERNODE);
     argsman.AddArg("-masternodeblsprivkey=<hex>", "Set the masternode BLS private key and enable the client to act as a masternode", ArgsManager::ALLOW_ANY | ArgsManager::SENSITIVE, OptionsCategory::MASTERNODE);
     argsman.AddArg("-platform-user=<user>", "Set the username for the \"platform user\", a restricted user intended to be used by Dash Platform, to the specified username.", ArgsManager::ALLOW_ANY, OptionsCategory::MASTERNODE);
+    argsman.AddArg("-deprecated-platform-user", "Let use -platform-user since it is deprecated from v21", ArgsManager::ALLOW_ANY, OptionsCategory::MASTERNODE);
```

Comment:
> Why don't we just drop -platform-user instead of deprecating it? Why start Evo deployment with some already outdated logic?

this feature has been become publicly available long as v0.16 of Dash Core. Even if our Evo is not using it, someone 3rd party in theory can use it. Should we ignore this possibility? That's a feature that exist more than 3 years